verl/trainer/fsdp_sft_trainer.py

```diff
@@ -20,6 +20,8 @@
 
 import os
 
+from verl.utils.config import config_normalize_batch_size
```

Comment:
Can you move this import above L35, which would be tidier

Reply:
(will do that in the next batch after discussions of other things)